forch/faucet_state_collector.py

```diff
@@ -1378,6 +1379,11 @@ def _get_port_attributes(self, switch, port):
                 ret_attr['type'] = 'egress'
                 return ret_attr
 
+            sequester_port_desc = self._config.orchestration.sequester_config.port_description
+            if sequester_port_desc and sequester_port_desc in port_info.get('description', ''):
```

Comment:
this seems unstable (e.g. 'tap' would also match 'taper') -- likely should be looking for equality

Reply:
Done